__tests__/users/email.test.ts

```diff
@@ -0,0 +1,33 @@
+import sendEmail from "../../src/utils/email";
+import userModel from "../../models/user.model";
+
+describe("sendEmail", () => {
+  it("should send email", async () => {
+    jest.setTimeout(30000);
+    const token = new userModel().generateAuthToken();
+    const email = "test@test.com";
+    const res = await sendEmail(email, token);
+    console.log(res);
```

Comment:
console.log do usunięcia

Reply:
jw. - to nic nie daje. w pewnym momencie pojawiło mi się jeszcze więcej błędów bez niego.